tests/unit/references/test_references_processors.py

```diff
@@ -63,12 +65,26 @@ def test_split_refextract_handles_unicode():
     assert expected == result
 
 
-def test_is_arxiv():
-    pass
+def test_is_arxiv_new_identifier():
+    result = _is_arxiv('arXiv:1501.00001v1')
```

Comment:
so `_is_arxiv`, confusingly, does not return a `bool`? maybe the name of the function should be changed then.